app/code/core/Mage/Eav/Model/Entity/Type.php

```diff
@@ -99,24 +99,33 @@ public function loadByCode($code)
     /**
      * Retrieve entity type attributes collection
      *
-     * @param   int $setId
-     * @return  Mage_Eav_Model_Resource_Entity_Attribute_Collection
+     * @param int|null $setId
+     * @param bool $useCache reuse local cache for collection
+     * @return Mage_Eav_Model_Resource_Entity_Attribute_Collection
      */
-    public function getAttributeCollection($setId = null)
+    public function getAttributeCollection(int $setId = null, bool $useCache = true)
```

Comment:
changing the signature breaks overridden classes, should we do something about that or we consider it ok since it's v20?

Reply:
Oh yeah my bad, it was just reflex adding the types to parameters. I will remove them.